docs/maven-example.md

```diff
@@ -0,0 +1,56 @@
+# Fabric8 Maven Plugin + Kompose:
+
+Lets deploy a Springboot Java application with Docker Compose file using fabric8 maven plugin to Kubernetes/Openshift.
+
+__1. Clone the example project from github__
+```bash
+$ git clone https://github.com/piyush1594/Kompose-Example.git
+```
+Navigate (cd) to `Kompose-Example` directory.
+
+__2. Install the kompose through maven plugin__ 
+```
+$ mvn fabric8:install
+```
+This command installs the `kompose` and other binaries on the host.
```

Comment:
what other binaries? link?

Reply:
this command: `mvn fabric8:install` has no description... What are we doing here? I believe the sentence "This command installs.." should be above it. Also needs to be changed to "This command installs Fabric8 and Kompose" or something similar.